trends_data/src/commonMain/kotlin/net/thechance/mena/trends/data/dto/UserInfoDto.kt

```diff
@@ -0,0 +1,18 @@
+package net.thechance.mena.trends.data.dto
+
+
+import kotlinx.serialization.SerialName
+
+import kotlinx.serialization.Serializable
+@Serializable
```

Comment:
could we remove this space 

```suggestion
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable